src/hipscat/catalog/partition_info.py

```diff
@@ -43,6 +43,15 @@ def get_highest_order(self) -> int:
 
         return highest_order
 
+    def write_to_file(self, catalog_base_dir: FilePointer):
```

Comment:
This function needs unit tests specifically for it.

Reply:
Created a [test case](https://github.com/astronomy-commons/hipscat/blob/62affea07125e0eb32fa3596aa25d0be4cb8231a/tests/hipscat/catalog/test_partition_info.py#L56) that writes the partition information to disk, reads it back, and ensures the data frames are similar.